lib/miq_automation_engine/engine/miq_ae_executor_body.rb

```diff
@@ -0,0 +1,135 @@
+class AutomateMethodException < StandardError
+end
+
+begin
+  require 'date'
+  require 'rubygems'
+  require 'active_support/all'
+  require 'socket'
+  Socket.do_not_reverse_lookup = true  # turn off reverse DNS resolution
+
+  require 'drb'
+  require 'yaml'
+
+  Time.zone = 'UTC'
+
+  MIQ_OK    = 0
+  MIQ_WARN  = 4
+  MIQ_ERROR = 8
+  MIQ_STOP  = 8
+  MIQ_ABORT = 16
+
+  DRbObject.send(:undef_method, :inspect)
+  DRbObject.send(:undef_method, :id) if DRbObject.respond_to?(:id)
+
+rescue Exception => err
+  STDERR.puts('The following error occurred during inline method preamble evaluation:')
+  STDERR.puts("  #{err.class}: #{err.message}")
+  STDERR.puts("  #{err.backtrace.join("\n")}") unless err.kind_of?(AutomateMethodException)
+  raise
+end
```

Comment:
No, that's the `$evm.log` one now down on 119. This one was always wrapped only around the preamble.. though it did encapsulate the DRb connection -- which is why I copied it to 66.